packages/api/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import express from 'express';
+import asyncHandler from 'express-async-handler';
+import serverlessExpress from '@codegenie/serverless-express';
+import bodyParser from 'body-parser';
+import path from 'path';
+import { getConfig } from './config';
+
+const getApp = async () => {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	const config = await getConfig();
+
+	const app = express();
+	const apiRouter = express.Router();
+
+	app.use(bodyParser.json({ limit: '40mb' }));
+
+	apiRouter.get(
+		'/healthcheck',
+		asyncHandler(async (req, res) => {
+			res.send('It lives!');
+		}),
+	);
+
+	app.use('/api', apiRouter);
+
+	if (process.env['AWS_EXECUTION_ENV'] !== undefined) {
+		app.use(express.static('frontend'));
+		app.get('/*', (req, res) => {
+			res.sendFile(path.resolve(__dirname, 'frontend', 'index.html'));
+		});
+	}
+
+	return app;
+};
+
+let api;
+if (process.env['AWS_EXECUTION_ENV'] !== undefined) {
```

Comment:
if possible maybe good to store this in a variable `runnningOnAws` or similar - or rely on the stage tag